apps/controllerx/cx_devices/ikea.py

```diff
@@ -574,3 +574,35 @@ def get_zha_actions_mapping(self) -> DefaultActionsMapping:
             "stop": Light.RELEASE,
             "release": Light.RELEASE,
         }
+
+ class W2049MediaPlayerController(MediaPlayerController):
+    def get_z2m_actions_mapping(self) -> DefaultActionsMapping:
+        return {
+            "on": MediaPlayer.CLICK_VOLUME_UP,
```

Comment:
Do you think that CLICK_VOLUME_UP makes more sense than PLAY_PAUSE? I do not see any PLAYY_PAUSE in the mapping and in my opinion, it is important to have that in a Media Player controller mapping.